tests.sh

```diff
@@ -184,6 +184,9 @@ test_eval 'timediff("02:00:00", "03:30:00")' '1.5'
 test_eval 'timediff("03:30:00", "02:00:00")' '22.5'
 test_eval 'timediff("02:00:00", "02:00:30")' '0.01'
 
+current_time=$(echo `date +"%H:%M:%S"`)
+test_eval 'current_time()' "\"$current_time\""
```

Comment:
this is flaky, although th likelihood of failing is very low. i thought about testing it by using timediff and checking if it's 0 or 1 but i don't think that'd be better

Reply:
Could also be tested with a regex, using egrep

Reply:
i see, to check if it fits the pattern, instead of checking if it displays the current_time. that is, in fact, a possibility

Reply:
i decided to leave this test and add one with the regex match